app/post.go

```diff
@@ -887,7 +888,8 @@ func (a *App) DoPostAction(postId, actionId, userId, selectedOption string) *mod
 	var httpClient *http.Client
 	url, _ := url.Parse(action.Integration.URL)
 	siteURL, _ := url.Parse(*a.Config().ServiceSettings.SiteURL)
-	if (url.Hostname() == "localhost" || url.Hostname() == "127.0.0.1" || url.Hostname() == siteURL.Hostname()) && strings.HasPrefix(url.Path, "/plugins") {
+	subpath, _ := utils.GetSubpathFromConfig(a.Config())
+	if (url.Hostname() == "localhost" || url.Hostname() == "127.0.0.1" || url.Hostname() == siteURL.Hostname()) && strings.HasPrefix(url.Path, path.Join(subpath, "plugins")) {
```

Comment:
If `GetSubpathFromConfig()` fails, `subpath` will be `""`, and the `HasPrefix` check will fail. That being said, I think this is an edge case that indicates other things as broken, so your simplification of this check makes sense :)

Reply:
Yup. I didn't want that failing to cause the other cases that don't require it to fail.